pkg/configuration/configuration_test.go

```diff
@@ -0,0 +1,56 @@
+package configuration_test
+
+import (
+    "strings"
+    "testing"
+
+    "github.com/form3tech-oss/github-team-approver-commons/pkg/configuration"
+    "github.com/magiconair/properties/assert"
+    "github.com/stretchr/testify/require"
+)
+
+func Test_ReadConfiguration(t *testing.T) {
+    tt := []struct {
+        name     string
+        payload  string
+        expected configuration.Configuration
+    }{
+        {
```

Comment:
What do you think about storing the artefact in `testdata/approver.yaml`? 

I think we would gain two benefits from doing so:
1. easier to work with the yaml artefact as you can lint it and the editor can understand it
2. the test file is more readable, specially if we were to introduce unhappy paths as well.

Reply:
I agree with both of those advantages and I think in this case it would be better. I generally default to having test input as close to the test function as possible, but to be honest in a table test you loose this anyway!